scripts/update-size.js

```diff
@@ -6,13 +6,16 @@ const sizeLimit = require('size-limit');
 const root = path.join(__dirname, '..');
 const readmePath = path.join(root, 'README.md');
 const readme = fs.readFileSync(readmePath, 'utf-8');
+const sizeConfig = require('../.size-limit')[0];
 
-const pkg = require(path.join(root, 'package.json'));
-const file = path.join(root, pkg.browser[pkg.module]);
-
-sizeLimit(file, { ignore: ['react', 'prop-types'] }).then(bytes => {
+sizeLimit(sizeConfig.path, sizeConfig).then(bytes => {
   const weight = formatBytes(bytes.gzip);
 
+  if (weight > sizeConfig.limit) {
```

Comment:
I havent noticed it before, but you are comparing strings here, u should compare raw bytes (so you should convert `sizeConfig.limit` to them, or find a package that is able to compare those units).

Reply:
@Andarist I have used now [Bytes] module(https://www.npmjs.com/package/bytes) to compare sizes, earlier **format-bytes.js** was used i removed that since we have option to convert values in bytes module itself.After removing reduces size to **1.67KB to 1.63KB**